source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/login/index.tsx

```diff
@@ -1,18 +1,28 @@
 import * as React from 'react';
+import { compose } from 'recompose';
+import { withRouter, RouterProps } from 'react-router';
 
+import { requireNoAuth } from '@lib/auth';
 import Login from '@ui/components/login';
 
 export const pathName = '/login';
 
-export default class LoginRoute extends React.Component {
+class LoginRoute extends React.Component<RouterProps> {
   state = { data: {}, errors: {} };
   render() {
+    const { history } = this.props;
+
     return (
       <div>
         <h2>Login Route </h2>
 
-        <Login />
+        <Login afterLogin={() => history.push('/tasks')}/>
```

Comment:
I guess for now it's ok to redirect to task but eventually we should redirect to who ever call the login page to remember where you came from

Reply:
agreed, that's not a recorded requirement, but that would be good behavior to have